src/capi.ts

```diff
@@ -128,7 +128,8 @@ const capiEndpoint = (articleId: string, key: string): string => {
         'displayHint',
         'starRating',
         'commentable',
-        'liveBloggingNow'
+        'liveBloggingNow',
+        'lastModified'
```

Comment:
You'll need to update `capi.test.ts` with this field

Reply:
Not sure if i have to create a new field as the date created is dynamic

Reply:
It will always be "lastModified" in the request and then the response will be different. The test is expecting this string:

https://content.guardianapis.com/cities/2019/sep/13/reclaimed-lakes-and-giant-airports-how-mexico-city-might-have-looked?format=thrift&api-key=TEST_KEY&show-atoms=all&show-fields=headline%2Cstandfirst%2CbylineHtml%2CfirstPublicationDate%2CshouldHideAdverts%2CshouldHideReaderRevenue%2CdisplayHint%2CstarRating%2Ccommentable%2CliveBloggingNow%2ClastModified&show-tags=all&show-blocks=all&show-elements=all&show-related=true

Reply:
Thanks Jamie, how did you come up with that string?